docs/use/Set-Up-DeFi-Tools/index.md

```diff
@@ -0,0 +1,62 @@
+---
+id: set-up-defi-tools
+order: 12
+title: Set up DeFi Tools
+---
+
+## Setting up Metamask
+### What is Metamask?
+MetaMask is a browser extension that allows you to use Ethereum Protocols or `dApps` ("decentralized Apps") such as Tinlake right in your browser. Metamask holds your private keys, and once it's set up, you can connect it to the dApp in your browser and confirm your transactions in the dApp with Metamask. It also serves as a wallet for ETH and `ERC20 tokens`, such as DAI, TIN or DROP and allows you to transfer and receive such tokens.
+
+### How to install Metamask in your browser
+1. Go to https://metamask.io/download.html.
+2. Choose the browser you want to use.
+3. You will be directed to corresponding web store.
+4. Follow the steps there to add the extension to your browser.
+5. After setting up your wallet, Metamask should pop-up as a Fox icon ![](https://storage.googleapis.com/centrifuge-hackmd/upload_7dd07df8a7fc7d1b29743ec9535dbba7.png =20x) in your browser bar.
+
+### How to use Metamask with Tinlake
+- Connect your account.
+- Click on Connect (top right corner).
+- Choose the Metamask or the wallet you would like to connect.
+![](https://storage.googleapis.com/centrifuge-hackmd/upload_7ce191ec0d5455f0228b8e9779c54719.png)
+
+## Set up a Ledger Hardware Wallet
+### How to use Metamask with a hardware wallet, e.g. Ledger?
+A hardware wallet stores the user's private keys in a secure hardware device. To use an address/wallet stored on a hard ware wallet a user needs to be in physical posession of this hardware wallet. Popolar hardware wallets are e.g [Ledger](https://www.ledger.com/) or [Trezor](https://trezor.io/).
+
+#### Set up the Ledger
```

Comment:
Should this not be the same heading as the other sections? i.e.
### Set up the Ledger device